acceptance/bundle/test.toml

```diff
@@ -1,2 +1,6 @@
 Local = true
 Cloud = true
+
+[EnvMatrix]
+# The lowest Python version we support. Alternative to "uv run --python 3.10"
+UV_PYTHON = ["3.10"]
```

Comment:
Can we decouple Python change to test runner change?

Can we name this something more descriptive, like DEFAULT_PYTHON_VERSION?

Reply:
We have to call it UV_PYTHON so that uv will pick it up